### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by Zensavona** [dockyard/ember-easy-form#208]
Is this project dead? The last open issue is from May and as far as I can tell there's no way of using this with Ember 2. Are there any plans to update this in the near future, or am I missing something entirely?

**Comment by bcardarella** [comment]
Not dead! We are currently updating all of our Ember addons to Ember 2.x. So far we've done:

* ember-async-button
* ember-data-route
* ember-admin

The plan for what's next:

* ember-cli-proxy-fixtures
* ember-easy-form
* ember-validations

This library is technically a blocker for Ember 1.10+, not just Ember 2.x mostly due to how this library was extending Handlebars and that changed quite a bit around 1.10. To that end, @diogomafra has generously offered to submit a PR to get to Ember 1.13, which will allow upgrades through the deprecation squashing strategy: https://twitter.com/diogomafra/status/658323551867531264

As far as 2.x goes, the feature we've been waiting on was recently merged into Ember and I think is targeted for 2.3(ish?) currently behind a feature flag.

Apologies for the time its taken to get here, unfortunately it was the nature of the beast when we were getting involved with private API. Sorry!

**Comment by Zensavona** [comment]
Thanks for the fast reply @bcardarella, great to hear that! Best of luck with the changes :)

**Comment by musaffa** [comment]
@bcardarella Why aren't you pulling the changes in the stable branch to the master?
